src/test/vscodeUiTest/suite/projectGenerationTest.ts

```diff
@@ -528,7 +528,7 @@ describe('Project generation tests', function () {
     await wizard.next();
 
     const allQuickPickInfo: QuickPickItemInfo[] = await wizard.getAllQuickPickInfo();
-    const allLabels: string[] = allQuickPickInfo.map((info) => info.label);
+    const allLabels: string[] = allQuickPickInfo.filter(info => info.label != "").map((info) => info.label);
```

Comment:
If you make this change in `requestUtils.ts#removeDuplicateArtifactIds` instead of here, does that also fix the tests?

Reply:
I could give it a try. I still don't understand why this is required for GH actions to pass, and not locally, but I was clearly seeing empty string entries in the list.